### PR TITLE
Updates for parsnip #1162

### PR DESCRIPTION
**Pull request by topepo**
Updates for tidymodels/parsnip#1162

The remote is temporary. I want to test the new changes in parsnip but will remove when (if!) the dev version passes

### CODE REVIEW

File: tests/testthat/_snaps/parsnip-survival-standalone.md [hunk 1]
```diff
@@ -4,21 +4,21 @@
...
+      ! The object does not have class <Surv>.
 
 # .check_cens_type()
 
     Code
       parsnip:::.check_cens_type(left_c, type = "right", fail = TRUE)
     Condition
       Error:
-      ! For this usage, the allowed censoring type is: 'right'
+      ! For this usage, the allowed censoring type is right.
```
Comment (simonpcouch) [inline tests/testthat/_snaps/parsnip-survival-standalone.md:15]: Ah, these need a `.val` or similar to make clear they're argument values.

Comment (topepo) [reply tests/testthat/_snaps/parsnip-survival-standalone.md:15]: Can you change that in parsnip and I'll do updates here?

Comment (simonpcouch) [reply tests/testthat/_snaps/parsnip-survival-standalone.md:15]: Done!

Comment (topepo) [reply tests/testthat/_snaps/parsnip-survival-standalone.md:15]: Danke

**Comment by simonpcouch** [comment]
Just sent in another round of changes after Emil's review!

**Comment by simonpcouch** [comment]
FYI, now have local changes here!

**Pull request merged**